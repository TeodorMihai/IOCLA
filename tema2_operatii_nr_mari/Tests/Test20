A1B2C3D4E5F600001234 * -432100006F5E4D3C2B1A
