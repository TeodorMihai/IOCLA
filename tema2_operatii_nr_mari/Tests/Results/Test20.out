34120000F6E5D4C3B2A1
1A2B3C4D5E6F00002143
489540EB91EDD2912571AE020C1AF93C4CA1662A
-2A66A14C3CF91A0C02AE712591D2ED91EB409548
